build-system/tasks/performance/measure-documents.js

```diff
@@ -70,6 +74,35 @@ const setupMeasurement = page =>
     });
   });
 
+/**
+ * Handles request interception
+ *
+ * @param {Request} interceptedRequest
+ * @param {Array<number>} endTimes
+ */
+async function handleRequests(interceptedRequest, endTimes) {
+  const interceptedUrl = interceptedRequest.url();
+  const matchArray = interceptedUrl.match(CDN_ANALYTICS_REGEXP);
+  if (matchArray) {
+    const filename = matchArray[1];
+    const pathToJson = path.join(
+      'extensions/amp-analytics/0.1/vendors/',
+      filename
+    );
+    const jsonString = await getFile(pathToJson);
+    interceptedRequest.respond({
+      status: 200,
+      contentType: 'application/json; charset=utf-8',
+      body: jsonString,
+    });
+  } else if (interceptedUrl.includes(ANALYTICS_PARAM)) {
+    endTimes.push(Date.now());
+    interceptedRequest.abort();
```

Comment:
Could you help point out where are we waiting for `interceptedRequest` to abort, and what is the timeout?

Reply:
We are aborting because we don't want the request ping to actually hit their endpoint. And the timeout is set by `networkidle0`.

Does that answer your questions?

Reply:
I see. That makes sense thanks.